src/hooks/useLanguage/useLanguage.test.ts

```diff
@@ -22,6 +23,8 @@ describe("useLanguage", () => {
   const getItem = jest.spyOn(window.localStorage.__proto__, "getItem");
 
   beforeEach(() => {
+    // @ts-ignore
+    languageConstants.TEMP_SUPPORTED_LANGUAGES = languageConstants.LANGUAGES;
```

Comment:
For the sake of the test, it makes sense to replace TEMP_SUPPORTED_LANGUAGES with all languages. Otherwise none of the test cases will be valuable since the only supported language is `python`